### DISCUSSION TITLE
Dataset resource

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#876] [Q&A] [answered]
Ada yang punya dataset atau sumber dataset history chat grup wa?

Kayak gini:
<img width="672" alt="Screenshot 2024-06-13 at 5 10 17 AM" src="https://github.com/teamalgoritma/community/assets/134937547/734a44ff-592a-48ce-8c0d-c5c6664a5ec7">

**Comment by ramdhinafinita** [comment]
Dear @ardianthescientist 

Pertanyaan Anda telah kami terima dan sedang kami usahakan terkait jawabannya. Mohon kesediaannya untuk menunggu, terima kasih.

Best regards,
Team Algoritma

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @ardianthescientist , terimakasih telah menunggu.

Untuk history chat group secara spesifik yang anda inginkan tidak tersedia. Namun, anda bisa coba untuk merujuk langkah export berikut dan melakukan cleansing sesuai dengan cara berikut:
https://github.com/tusharnankani/whatsapp-chat-data-analysis/blob/main/whatsapp-data-analysis.ipynb

untuk code bagian split_formats dan datetime_formats. Anda bisa sesuaikan dengan penulisan format pada chat yang berhasil di export. 

Format export.txt akan berbeda, berikut contoh hasil yang kami coba:
![image](https://github.com/teamalgoritma/community/assets/64061969/7a9d6cb4-63af-421d-9694-89fefa3b0716)


Tampilan akan seperti berikut:
![image](https://github.com/teamalgoritma/community/assets/64061969/d2dd6601-f197-4f37-8a5a-c029b93d5717)

Jika jawaban ini membantu, silahkan klik mark as answer. Terimakasih
